# This file is used for describing the corridor in this task.
# start point
start_pt:
  pos: [-1.0, 1.0, -1.2]
  vel: [0.0, 0.0, 0.0]
  acc: [0.0, 0.0, 0.0]
# end point 
end_pt:
  pos: [-0.8,-1.0,-1.0]
  vel: [0.0, 0.0, 0.0]
  acc: [0.0, 0.0, 0.0]
# corridor 
num_polyhedron: 3
corridor:
  - s_scale: 20.0
    upper_vel: [1.0,1.0,1.0]
    lower_vel: [-1.0, -1.0, -1.0]
    upper_acc: [1.0,1.0,1.0]
    lower_acc: [-1.0, -1.0, -1.0]
    A_pos: 
      row: 3
      col: 3
      data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0,0.0, 0.0, 1.0]
    b_u_pos: [1.80, 2.12, -0.20]
    b_l_pos: [-1.21, 0.97, -4.0]
  - s_scale: 20.0
    upper_vel: [1.0,1.0,1.0]
    lower_vel: [-1.0, -1.0, -1.0]
    upper_acc: [1.0,1.0,1.0]
    lower_acc: [-1.0, -1.0, -1.0]
    A_pos: 
      row: 3
      col: 3
      data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0,0.0, 0.0, 1.0]
    b_u_pos: [1.86, 1.73, -0.2]
    b_l_pos: [0.80, -1.13, -4.0]
  - s_scale: 20.0
    upper_vel: [1.0,1.0,1.0]
    lower_vel: [-1.0, -1.0, -1.0]
    upper_acc: [1.0,1.0,1.0]
    lower_acc: [-1.0, -1.0, -1.0]
    A_pos: 
      row: 3
      col: 3
      data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0,0.0, 0.0, 1.0]
    b_u_pos: [1.85, -0.47, -0.2]
    b_l_pos: [-0.83, -1.47, -4.0]


